tests/fake-block-tracker.ts

```diff
@@ -13,10 +13,9 @@ export class FakeBlockTracker extends PollingBlockTracker {
     super({
       provider: new SafeEventEmitterProvider({ engine: new JsonRpcEngine() }),
     });
-  }
-
-  override async _start() {
```

Comment:
Did this not work anymore? It'd be nice not to use `as any` here.

Reply:
Hmm. I'm wondering if we can solve this the same way that I want to solve the [fake provider](https://github.com/MetaMask/core/blob/main/packages/network-controller/tests/fake-provider.ts#L85) — that is, instead of extending from a block tracker class, have a class that fulfills the block tracker interface. You're right that that would be a separate change though. This is fine for now — thanks for clarifying.